docs/flink-getting-started.md

```diff
@@ -634,6 +634,26 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+## UPSERT mode
```

Comment:
I feel it is better to put it in the same level of `###` as other two write modes